test/parser.ts

```diff
@@ -49,6 +49,12 @@ describe('SimpleParser', () => {
 			const output = [TEXT('あ'), EMOJI_CODE('bar'), TEXT('い')];
 			assert.deepStrictEqual(mfm.parseSimple(input), output);
 		});
+
+		test('U+FE0F', () => {
```

Comment:
```suggestion
		test('Ignore Variation Selector preceded by Unicode Emoji', () => {
```

こっちの方が趣旨をより説明していると思います 👀

Reply:
ありがとうございます。早速取り込みます。